modules/loader/src/main/java/com/opengamma/strata/loader/csv/FullSwapTradeCsvLoader.java

```diff
@@ -267,8 +267,11 @@ private static PeriodicSchedule parseAccrualSchedule(CsvRow row, String leg) {
     builder.endDate(LoaderUtils.parseDate(getValueWithFallback(row, leg, END_DATE_FIELD)));
     builder.frequency(Frequency.parse(getValue(row, leg, FREQUENCY_FIELD)));
     // adjustments
-    BusinessDayAdjustment dateAdj = parseBusinessDayAdjustment(row, leg, DATE_ADJ_CNV_FIELD, DATE_ADJ_CAL_FIELD)
-        .orElse(BusinessDayAdjustment.NONE);
+    BusinessDayAdjustment dateAdj = parseBusinessDayAdjustmentWithFallback(
```

Comment:
Would it not be necessary to update all uses of `parseBusinessDayAdjustment`, not just this one?

Reply:
We should preserve the logic for the other ones: startDate & endDate.